src/components/HotRepositories.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useState } from "react";
+import { FaArrowAltCircleUp } from "react-icons/fa";
+import { VscIssues } from "react-icons/vsc";
+import { AiOutlineStar } from "react-icons/ai";
+import { BiGitPullRequest } from "react-icons/bi";
+import { RiCheckboxCircleFill } from "react-icons/ri";
+
+const hotRepo = [
+  {
+    id: 0,
+    organization: "Oven",
+    orgImg: "https://avatars.githubusercontent.com/u/108928776?s=200&v=4",
+    name: "Bun",
+    description: "Incredibly fast JavaScript runtime, bundler, transpiler and package manager.",
+    upvoted: false,
+    issues: "503",
+    stars: "27.7k",
+    PR: "262",
+    img: [
+      "https://avatars.githubusercontent.com/u/709451?s=64&v=4",
+      "https://avatars.githubusercontent.com/u/2148168?s=64&v=4",
+      "https://avatars.githubusercontent.com/u/70155278?s=64&v=4",
+      "https://avatars.githubusercontent.com/u/56601352?s=64&v=4",
+      "https://avatars.githubusercontent.com/u/790659?s=64&v=4",
+    ],
+  },
+  {
+    id: 1,
+    organization: "Pocketbase",
+    orgImg: "https://avatars.githubusercontent.com/u/101000011?s=200&v=4",
+    name: "Pocketbase",
+    description: "Open Source realtime backend in 1 file",
+    upvoted: false,
+    issues: "72",
+    stars: "7.3k",
+    PR: "32",
+    img: [
+      "https://avatars.githubusercontent.com/u/8248071?s=64&v=4",
+      "https://avatars.githubusercontent.com/u/43366254?s=64&v=4",
+      "https://avatars.githubusercontent.com/u/38179369?s=64&v=4",
+      "https://avatars.githubusercontent.com/u/26606825?s=64&v=4",
+      "https://avatars.githubusercontent.com/u/8593614?s=64&v=4",
+    ],
+  },
+  {
+    id: 2,
+    organization: "Open-Sauced",
+    orgImg: "https://avatars.githubusercontent.com/u/57568598?s=200&v=4",
+    name: "Open-Sauced",
+    description: " This is a project to identify your next open source contribution.",
+    upvoted: false,
+    issues: "293",
+    stars: "726",
+    PR: "1k",
+    img: [
+      "https://avatars.githubusercontent.com/u/5713670?s=64&v=4",
+      "https://avatars.githubusercontent.com/u/237133?s=64&v=4",
+      "https://avatars.githubusercontent.com/u/11777161?s=64&v=4",
+      "https://avatars.githubusercontent.com/u/14043845?s=64&v=4",
+      "https://avatars.githubusercontent.com/u/22990146?s=60&v=4",
+    ],
+  },
+];
+
+const HotRepositories = () => {
+  const [hotRepos, setHotRepos] = useState(hotRepo);
+
+  const handleVoted = (id: number) => {
+    const votedIdx = hotRepos.findIndex((obj) => obj.id == id);
+    hotRepos[votedIdx].upvoted = !hotRepos[votedIdx].upvoted;
+
+    setHotRepos([...hotRepos]);
+  };
```

Comment:
note: We have a function to make this work. This placeholder does not follow that pattern. Despite this being mocked data, we could still makes this work live.

Reply:
Could you please share more details on the function? 

I could take a look and make the needed changes

Reply:
https://github.com/open-sauced/hot/blob/main/src/components/PostGrid.tsx#L25-L30

Reply:
Let's not implement that in this pr, since the scope is creeping.